Automattic/wp-calypso
# My Home: update video icon on learn and grow section

#### Changes proposed in this Pull Request

* Update the video icon on the Learn and Grow section of My Home using the Material play icon instead of an outdated film strip.

Before | After
------------ | -------------
<img width="444" alt="image" src="https://user-images.githubusercontent.com/448298/84397590-cda45b00-abcc-11ea-8485-3ac4eaee315d.png"> | <img width="446" alt="image" src="https://user-images.githubusercontent.com/448298/84397571-c5e4b680-abcc-11ea-8551-2a272c4fd370.png">

#### Testing instructions



* Checkout this branch or use the calypso.live link
* Visit My Home on any site
* Confirm the video icon in the Learn and Grow section has been updated

Fixes #43170
